docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -86,6 +86,38 @@ The `-e` option is to ensure the official Azure CLI package is installed. This c
 
 You can now run the Azure CLI with the `az` command from either Windows Command Prompt or PowerShell.
 
+## Enable Tab Completion on PowerShell
+
+When typing in a command followed by a parameter, pressing the tab key will display a list of available options for that parameter. This saves you time and effort in remembering specific arguments or looking them up in the reference documentation.  This capability applies to commands and parameters and also extends to values such as account name and subscription name.
```

Comment:
On Bash, the first <kbd>Tab</kbd> key press auto-completes if there is only one available option. Otherwise, the second <kbd>Tab</kbd> key press shows all available options. Is this the same for PowerShell? If so, we need to make the doc more accurate.

Reply:
I use Zsh as default shell, the PowerShell behavior is similar to Zsh.

This is a brief introduction, I think we don't need to go into the details.